bbc_feeds/__init__.py

```diff
@@ -34,20 +34,23 @@ def tech(self, limit=False):
 
     def science(self, limit=False):
         return(parse(self.url + '/science_and_environment/rss.xml', limit=limit))
-
-    def culture(self, limit=False):
-        return(parse('https://www.bbc.com/culture/feed.rss', limit=limit))
+    
+    def top_stories(self, edition='int', limit=False):
+        return(parse(self.url + f'/rss.xml?edition={edition}', limit=limit))
 
 class sports():
     def __init__(self):
-        self.url = 'http://feeds.bbci.co.uk/sport'
+        self.url = 'https://feeds.bbci.co.uk/sport'
 
     def all(self, limit=False):
         return(parse(self.url + '/rss.xml', limit=limit))
 
     def cricket(self, limit=False):
         return(parse(self.url + '/cricket/rss.xml', limit=limit))
 
+    def football(self, limit=False):
```

Comment:
I think we shouldn't rename the `soccer` function because `football` might be misleading toward American football (turns out BBC actually does have an American football page at https://www.bbc.com/sport/american-football). I know many people prefer to refer to the sport as "football" but since "football" has two definitions and "soccer" has one, I think it's best to keep it to unambiguous with "soccer".